portfolio/src/main/webapp/script.js

```diff
@@ -176,6 +183,14 @@ function disableSubmit(elementId) {
   document.getElementById(elementId).disabled = true;
 }
 
+/**
+ * Add a text element that instructs the user to log in to view comments.
+ */
+function addCommentLoginInstruction() {
+  const commentContainer = document.getElementById('comment-container');
+  commentContainer.innerText = 'Please log in to view the comments.';
+}
+
 /**
  * Check if the correlated input element box has text; if so, enable the 
  * "submit" button corresponding to that input element.
```

Comment:
request: I think it'd be clearer if the submit button was disabled while you're logged out as well. I found it surprising that the button was clickable but my comments weren't actually displayed while I was logged out.

Reply:
![image](https://user-images.githubusercontent.com/7987279/84531991-4a513b00-ac9a-11ea-9172-69b83a5a2996.png)
I have it set up such that you can't see any comments, or any comment-related material, when the user is logged out -- is this not what you are seeing?

Reply:
huh, yea that's different from what I see: https://screenshot.googleplex.com/dUoBmXtWXLw I highlighted the 'comments-container' div since that seems like it may not contain the correct contents.

I'm pretty sure the code I'm using is up to date with this repo. Do you have any local js or html changes which haven't been pushed?

Reply:
Fixed! looks good to me, go ahead with the merge.